front_end/models/react_native/FuseboxBindingsModel.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import * as SDK from '../../core/sdk/sdk.js';
+import type * as Common from '../../core/common/common.js';
+
+type JSONValue = null | string | number | boolean | {[key: string]: JSONValue} | JSONValue[];
+type DomainName = 'react-devtools';
+type DomainMessageListener = (message: JSONValue) => void;
+type BindingCalledEventTargetEvent = Common.EventTarget.EventTargetEvent<SDK.RuntimeModel.EventTypes[SDK.RuntimeModel.Events.BindingCalled]>;
+
+const RUNTIME_GLOBAL = '__FUSEBOX_REACT_DEVTOOLS_DISPATCHER__';
+
+export class FuseboxBindingsModel extends SDK.SDKModel.SDKModel {
+  private readonly domainToListeners: Map<DomainName, Set<DomainMessageListener>> = new Map();
+
+  constructor(target: SDK.Target.Target) {
+    super(target);
+
+    this.initialize(target);
+  }
+
+  private initialize(target: SDK.Target.Target): void {
```

Comment:
Sidenote: is there a style-guide in this repo for TS build-time `private` vs JS runtime private `#`?

Reply:
No, I don't think so, since its TS, then I think we should use `private` by default